development/compiling/compiling_with_mono.rst

```diff
@@ -14,8 +14,16 @@ Requirements
 - pkg-config
 
 You may need to import necessary certificates for NuGet to perform HTTPS
-requests. You can do this with the following command (on Windows, you can run it
-from the Mono command line prompt)::
+requests.
+
+First you have to download **curl**'s CA (Certificate Autorities) certificate bundle: https://curl.haxx.se/ca/cacert.pem
+
+Then you run the following command to import it (on Windows, you can run it
+from the Mono command line prompt, or from the regular command prompt if you added the Mono installation's bin directory to the PATH environemnt variable)::
+
+    cert-sync --user cacert.pem
+
+Aternatively, you can do this with the following command, though it won't work for Mono v6 or newer ::
```

Comment:
Is there any info somewhere to confirm mozroots no longer works with 6.0.0? If not, I would prefer to say `Alternatively, you can use the following command, though it's deprecated and may not work ::`

Also typo `Aternatively` -> `Alternatively`.

Reply:
It seems recent nuget versions requires a new certificate storage format that mozroots doesn't support (see #32284)

![image](https://user-images.githubusercontent.com/10672233/66172058-e792a480-e620-11e9-9f21-ec4e4db9410c.png)